synapse/app/homeserver.py

```diff
@@ -431,6 +431,8 @@ def phone_stats_home():
 
         daily_sent_messages = yield hs.get_datastore().count_daily_sent_messages()
         stats["daily_sent_messages"] = daily_sent_messages
+        stats["cache_factor"] = float(os.environ['SYNAPSE_CACHE_FACTOR'])
```

Comment:
this will (I think) barf if the env var isn't set. Can you just `import CACHE_SIZE_FACTOR from synapse.util.caches` ?